templates/common/bin/pxe-init.sh

```diff
@@ -64,7 +64,9 @@ fi
 
 # Build an ESP image
 pushd /var/lib/ironic/httpboot
-if [ ! -a "esp.img" ]; then
+if ! command -v dd 2>&1 /dev/null || ! command -v mkfs.msdos 2>&1 /dev/null || ! command -v mmd 2>&1 /dev/null; then
```

Comment:
The redirect to /dev/null - it needs to be `2>&1 > /dev/null` 

However, I think we can actually just let it do normal output, no redirect STDERR to STDOUT and no redirect to /dev/null i.e : `if ! command -v dd || ! command -v mkfs.msdos || ! command -v mmd; then`